src/scrapers/base-scraper.ts

```diff
@@ -125,6 +125,12 @@ export interface ScraperOptions {
    * Options for manipulation of output data
    */
   outputData?: OutputDataOptions;
+
+  /**
+   * Perform additional operation for each transaction to get more information about it (Like category).
+   * Please note, it will take more time to finish the process.
+   */
+  extraScrap?: boolean;
```

Comment:
consider following the namespace concept, maybe 
```{
scraping: {
  additionalTransactionInformation: boolean
}
}
```

Reply:
Fixed